static/sass/developer-include-sidebar.scss

```
@@ -0,0 +1,230 @@
+// offset in conjunction with headroom js
+.documentation main aside.menu {
+    padding-top: 6.5rem !important;   
+}
+
+.documentation header.headroom--unpinned + main aside.menu {
+    padding-top: 1.5rem !important;  
+}
+
+// sidebar width is fixed for desktop
+$sidebarWidth: 17.5vw;
+
+.menu-wrap {
+    padding: 0 1.5vw !important;
+    z-index: 665;
+}
+
+.menu-wrap + article.content {
+    padding-left: calc($sidebarWidth + 0.75rem);
+}
+
+// sidebar layout
+.documentation aside.menu {
+    display: flex;
+    flex-direction: column;
+    min-height: 100%;
+    border: none !important;
+    padding-bottom: 8rem !important;
+    padding-top: 0 !important;
+    background: transparent none !important;
+    overflow-x: hidden;
+    overflow-y: scroll;
```

```suggestion
    overflow-y: auto;
```
Maybe adding a scroll gutter instead would be cleaner I think.